cmd/kubelet/app/server.go

```diff
@@ -174,13 +218,88 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 		},
 	}
 
+	// Flags registered here only produce help text. Flag parsing is handled privately in Run, above.
```

Comment:
not a fan of "for help only" constructs... this is likely to drift.

if you need to disable flag parsing to get the full set of args for reparsing, can we:
1. leave the config struct and flag registration as it was (ensures help accurately reflects what actually gets used when running)
2. do `cmd.Flags().Parse(args)` as the first step in Run()
3. continue, doing the `config file load/reparse` and `dynamic config load/reparse` blocks as needed

Reply:
I think that can also work.

Reply:
Done